downstream/modules/platform/ref-ldap-config-on-pah.adoc

```diff
@@ -0,0 +1,177 @@
+auth_ldap_group_type_class= 'django_auth_ldap.config:GroupOfNamesType'
+-----
+
+====
+
+. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, super user access, mirroring, or others), proceed to the next section.
+
+
+== Configuring extra LDAP parameters
+
+If you plan to set up super user access, user groups, mirroring or other extra parameters, you can create a yaml file that comprises them in your `ldap_extra_settings` dictionary.
+
+.Procedure
+
+. Create a yaml file that will contain `ldap_extra_settings`, such as the following:
++
+----
+#ldapextras.yml   
+---
+ldap_extra_settings:
+  AUTH_LDAP_USER_ATTR_MAP: '{"first_name": "givenName", "last_name": "sn", "email": "mail"}'
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use this example to set up a superuser flag based on membership in an LDAP group.
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_USER_FLAGS_BY_GROUP: {"is_superuser": "cn=pah-admins,ou=groups,dc=example,dc=com",}
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use this example to set up superuser access.
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_USER_FLAGS_BY_GROUP: {"is_superuser": "cn=pah-admins,ou=groups,dc=example,dc=com",}
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use this example to mirror all LDAP groups you belong to.
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_MIRROR_GROUPS: True
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use this example to map LDAP user attributes (such as first name, last name, and email address of the user).
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_USER_ATTR_MAP: {"first_name": "givenName", "last_name": "sn", "email": "mail",}
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use the following examples to grant or deny access based on LDAP group membership.
+.. To grant {PrivateHubName} access (for example, members of the `cn=pah-nosoupforyou,ou=groups,dc=example,dc=com` group): 
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_REQUIRE_GROUP: 'cn=pah-users,ou=groups,dc=example,dc=com'
+...
+----
+.. To deny {PrivateHubName} access (for example, members of the `cn=pah-nosoupforyou,ou=groups,dc=example,dc=com` group):
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_DENY_GROUP: 'cn=pah-nosoupforyou,ou=groups,dc=example,dc=com'
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use this example to enable LDAP debug logging.
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  GALAXY_LDAP_LOGGING: True
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
++
+[NOTE]
```

Comment:
1) Given the criticality of this piece of info., do you think it'll be a good idea to flag this info. as Important or Caution instead of a Note?
2) Does this step apply for specifically executing the `setup.sh -e @ldapextras.yml` command or for all instances for executing the setup.sh command? If it's for all instances,  consider changing the placement of this text to indicate that it applies for all instances.

Reply:
1. Per the Supplemental guide, we have 3 options (see [Admonitions](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions)): Note, Important, and Warning. IMHO, this note simply gives the customer an alternative if it's "not practical to re-run `setup.sh` and then states that this should be a last resort due to human error, but it doesn't say that your system will suffer significant damage if you do not do this action described in the note.
2. Are you referring to the note itself? Per the SME, Lucas Benedito, this only refers to this particular step.